backend/amd64/cfg_stack_operands.ml

```diff
@@ -194,7 +194,7 @@ let basic (map : spilled_map) (instr : Cfg.basic Cfg.instruction) =
   | Op (Intop (Ipopcnt | Iclz _| Ictz _))
   | Op (Move | Spill | Reload | Negf | Absf | Const_float _ | Compf _ | Stackoffset _
        | Load _ | Store _ | Name_for_debugger _ | Probe_is_enabled _
-       | Opaque | Begin_region | End_region )
+       | Valueofint | Intofvalue | Opaque | Begin_region | End_region )
```

Comment:
I am fine with the current code, but
actually wonder whether for
`Valueofint | Intofvalue | Opaque` we
could pretend there are no spilled
registers. No code will be emitted
for the operation anyway, and maybe
we could end up emitting useless
spill/reload instructions. I don't think
it should be changed in this PR, but
maybe leave a CR so that we can
discuss it later.

Reply:
yes, it would be better to fix it here.

Reply:
A similar concern was addressed for the "upstream" regalloc in reloadgen (treating the new operations similarly to Imove). The fix for `cfg` can be done in a separate PR.